include/Realm.h

```diff
@@ -455,6 +459,10 @@ class Realm {
   TurbulenceAveragingPostProcessing *turbulenceAveragingPostProcessing_;
   DataProbePostProcessing *dataProbePostProcessing_;
   Actuator *actuator_;
+  std::shared_ptr<ActuatorMetaFAST> actuatorMeta_;
```

Comment:
I think that's a great idea, it will simplify the pointer variables and help keep things neater.  If it's alright, I think we should do this after merging in this fixed wing code (maybe also along with any fixes to thrust computations), since that affects all of the actuator NGP codes.